src/pages/[platform]/build-a-backend/auth/add-social-provider/index.mdx

```diff
@@ -387,7 +387,7 @@ After configuring the OAuth endpoints (with Cognito Hosted UI), you can integrat
 
 A custom state is not required, but if you are looking to add one, you are able to do so by passing a string value (e.g. `signInWithRedirect({ customState: 'xyz' })`) and listening for the custom state via [Hub](/[platform]/build-a-backend/utilities/hub/). You can also use the Hub eventing system to catch errors using `signInWithRedirect()` to listen for the `signInWithRedirect_failure` event.
 
-<InlineFilter filters={["javascript"]}>
+<InlineFilter filters={["javascript", "angular", "nextjs", "react", "vue"]}>
```

Comment:
We might want to check with content on this one, this block is showing react specific code and there are lots of other places on the page that include all of the web platforms in the filter list so maybe this was intentional?

Reply:
I left in JS because it was there initially, but removed angular, nextjs, and vue